docs/CHANGELOG.rst

```diff
@@ -21,6 +21,7 @@ Added
 * Add ``upload-metabolic-pathway`` process
 * Add memory-optimized primerclip as a separate ``align-bwa-trim2`` process
 * Add ``workflow-accel-2`` workflow
+* Add R, tabix, and CheMut R library to ``resolwebio/dnaseq`` Docker image
```

Comment:
@tristanbrown This needs to move to the unreleased section of the changelog now. Please make the fix so we can merge this.

Reply:
Fixed.